src/test/scala/nl/knaw/dans/bag/v0/ManifestSpec.scala

```diff
@@ -269,14 +269,22 @@ class ManifestSpec extends TestSupportFixture with TestBags with BagMatchers wit
     }
   }
 
```

Comment:
I don't see a test for copying to an empty directory. Is that already covered somewhere else?

Reply:
Only with submission test in deposit-api. I know, not good enough here.